members/models.py

```diff
@@ -313,11 +313,16 @@ def save(self, *args, **kwargs):
         return super(ActivityParticipant, self).save(*args, **kwargs)
 
 class Volunteer(models.Model):
-    member = models.ForeignKey(Member)
+    class Meta:
+        verbose_name = "Frivillig"
+        verbose_name_plural = "Frivillige"
+    member = models.ForeignKey(Person)
     department = models.ForeignKey(Department)
     def has_certificate(self):
-        return self.person.has_certificate
-    added = models.DateTimeField(auto_now_add=True, blank=True, editable=False)
+        return self.member.has_certificate
+    added = models.DateTimeField('Start', default=timezone.now)
+    removed = models.DateTimeField('Slut', blank=True, null=True, default=None)
+    approved = models.BooleanField('Godkendt af afdelingsleder',default=True)
```

Comment:
vi bruger altid dato felter i stedet for booleans - det er altid rart ud over yes/no at kunne se hvornår den har fået denne state :)